packages/keyring-eth-ledger-bridge/test/document.shim.ts

```diff
@@ -1,24 +1,31 @@
-// eslint-disable-next-line import/no-mutable-exports
+// eslint-disable-next-line import-x/no-mutable-exports
 let documentShim: any;
 
+type Element = {
```

Comment:
nit

```suggestion
type IframeElement = {
```

Reply:
I must say I do prefer `Element` to match the `createElement` name here, is that ok for you?

Reply:
is just a nit, not going to die on this hill